run.py

```diff
@@ -11,6 +13,6 @@
         host = host_val
     else:
         host = '127.0.0.1'
-    application.run(host=host, port=5050, threaded=True)
+    freezer.run(host=host, port=5050, threaded=True)
```

Comment:
We want to be able to preserve the ability to run either the application dynamically or to freeze it. This may involve using something like Flask-Script to allow us to create  commands. See https://github.com/USGS-CIDA/WQP_UI/blob/master/manage.py for an example of how we are doing this for other flask tools. I